package.json

```diff
@@ -33,9 +33,9 @@
     "test:profiler": "tap \"packages/dd-trace/test/profiling/**/*.spec.js\"",
     "test:profiler:ci": "npm run test:profiler -- --coverage --nyc-arg=--include=\"packages/dd-trace/src/profiling/**/*.js\"",
     "test:integration": "mocha --colors --timeout 30000 \"integration-tests/*.spec.js\"",
-    "test:integration:cucumber": "mocha --colors --timeout 30000 \"integration-tests/cucumber/*.spec.js\"",
-    "test:integration:cypress": "mocha --colors --timeout 30000 \"integration-tests/cypress/*.spec.js\"",
-    "test:integration:playwright": "mocha --colors --timeout 30000 \"integration-tests/playwright/*.spec.js\"",
+    "test:integration:cucumber": "mocha --colors --timeout 30000 --exit \"integration-tests/cucumber/*.spec.js\"",
```

Comment:
Why is this needed? Needing this means the test isn't exiting properly and Mocha needs to kill it. Not only is this working around the issue, it won't be compatible with tap.

Reply:
yeah there was some kind of weird race condition where if the tested ended before the cypress process was finished, a handle was kept alive, not allowing the process to finish. I can't explain but [this change](https://github.com/DataDog/dd-trace-js/pull/3167/commits/6786b87b6090458c3c9c224b918a55ff9dae7ff1#diff-68748ac051251a008cc2ba15e7824343438d0be3f665d7463f031c07c7fb6baeR225-R234) seems to fix it